crates/rome_cli/tests/snap_test.rs

```diff
@@ -21,6 +21,20 @@ struct CliSnapshot {
     pub files: HashMap<String, String>,
     /// messages written in console
     pub messages: Vec<String>,
+    /// possible termination error of the CLI
+    pub termination: Option<Termination>,
+}
+
+impl CliSnapshot {
+    pub fn form_result(result: Result<(), Termination>) -> Self {
```

Comment:
This looks like a typo:
```suggestion
    pub fn from_result(result: Result<(), Termination>) -> Self {
```